### PR TITLE
Plasmaman paramedics now start with nitrile gloves

### PR DESCRIPTION
**Pull request by Red-byte3D**
## About The Pull Request
Plasmaman paramedics now start with nitrile gloves like their counterpart (all other races)
## Why It's Good For The Game
mostly a qol update because i would assume the race that shouldnt be taking any clothes off would come to the shift prepared with the proper gloves 
## Changelog
:cl:
qol: Plasmamen paramedics now start with nitrile gloves
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
this makes paramedics the only plasmamen with not plasmaman gloves (other than captain)
add a subtype of plasmaman gloves with nitrile attributes like how plasmaman engineering does it

**Comment by SmArtKar** [comment]
Uh, yknow, this means that they will burst into flames as soon as they spawn because plasmamen gloves are also required for the suit to work propperly sooooo

**Comment by InsaneRed** [comment]
> Uh, yknow, this means that they will burst into flames as soon as they spawn because plasmamen gloves are also required for the suit to work propperly sooooo

no....

**Comment by InsaneRed** [comment]
> this makes paramedics the only plasmamen with not plasmaman gloves (other than captain)
> add a subtype of plasmaman gloves with nitrile attributes like how plasmaman engineering does it

done

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -86,6 +86,14 @@
 	icon_state = "blackplasma"
 	inhand_icon_state = "blackplasma"
 
+/obj/item/clothing/gloves/color/plasmaman/plasmanitrile
+	name = "nitrile envirogloves"
+	desc = "Pricy nitrile gloves made for plasmaman"
```
Comment (BootlegBow) [inline code/modules/clothing/gloves/miscellaneous.dm:91]: bit of a nitpick, but this should really be a plural 'plasmamen'

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 2]
```diff
@@ -86,6 +86,14 @@
 	icon_state = "blackplasma"
 	inhand_icon_state = "blackplasma"
 
+/obj/item/clothing/gloves/color/plasmaman/plasmanitrile
+	name = "nitrile envirogloves"
+	desc = "pricy nitrile gloves made for plasmamen"
```
Comment (Fikou) [inline code/modules/clothing/gloves/miscellaneous.dm:91]: descs should start capitalized it looks bad when they arent

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 3]
```diff
@@ -86,6 +86,14 @@
 	icon_state = "blackplasma"
 	inhand_icon_state = "blackplasma"
 
+/obj/item/clothing/gloves/color/plasmaman/plasmanitrile
+	name = "Nitrile envirogloves"
```
Comment (Mothblocks) [inline code/modules/clothing/gloves/miscellaneous.dm:90]:
```suggestion
	name = "nitrile envirogloves"
```

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 4]
```diff
@@ -86,6 +86,14 @@
...
+	desc = "Pricy nitrile gloves made for plasmamen"
```
Comment (Mothblocks) [inline code/modules/clothing/gloves/miscellaneous.dm:91]:
```suggestion
	desc = "Pricy nitrile gloves made for plasmamen."
```

**Pull request merged**